fvm_actor_utils/src/receiver/mod.rs

```diff
@@ -132,16 +147,16 @@ impl<T: RecipientData> ReceiverHook<T> {
             &TokenAmount::zero(),
         )?;
 
-        match receipt.exit_code {
+        match ret.0 {
```

Comment:
Ok, I think we want to name the return struct. This is going to get confusing.

Reply:
Done :)